src/actions/vis-state-actions.d.ts

```diff
@@ -204,6 +204,15 @@ export function showDatasetTable(
   dataId: string
 ): Merge<ShowDatasetTableUpdaterAction, {type: ActionTypes.SHOW_DATASET_TABLE}>;
 
+export type UpdateDatasetColorUpdater = {
+  dataId: string,
+  newColor: [number, number, number]
```

Comment:
There is a type fo hex color. you can import `HexColor` from `'vis-state-updater`